inst/rmd/umap_qc.rmd

```diff
@@ -0,0 +1,100 @@
+## Dimensionality Reduction 
+
+```{r message=FALSE}
+# create UMAP colored by number of genes detected 
+scater::plotUMAP(processed_sce,
+                 point_size = 0.3,
+                 point_alpha = 0.5,
+                 colour_by = "detected") +
+  scale_color_viridis_c()+
+  theme_bw() + 
+  guides(
+    color = guide_colorbar(title = "Number of \ngenes detected")
+  )
+```
+
+Compromised cells, calculated using `miQC` described above, are filtered and removed from the library prior to downstream analyses. 
+The raw counts from all remaining cells are then normalized prior to selection of highly variable genes and dimensionality reduction. 
+The above plot shows the UMAP (Uniform Manifold Approximation and Projection) embeddings for each cell, coloring each cell by the total number of genes detected per cell.
+The plots below show the same UMAP embeddings, coloring each cell by the expression level of the labeled gene.
```

Comment:
Should we make this a separate subsection? I think it is worth calling out as "Expression patterns of variable genes" or something like that?